src/visualization/types/Table/TableCell.tsx

```diff
@@ -272,12 +273,9 @@ class TableCell extends PureComponent<Props> {
       return (this.fieldName || '').toString()
     }
 
-    if (
-      !isNaN(+data) &&
-      decimalPlaces.isEnforced &&
-      decimalPlaces.digits < 100
-    ) {
-      return (+data).toFixed(decimalPlaces.digits)
+    if (!isNaN(+data)) {
+      // method needs the first arg to be a number to work properly
+      return formatStatValue(+data, {decimalPlaces})
```

Comment:
do you know what's with these casts to numbers? do you think it would be more idiomatic and communicative to do a `parseInt(data, 10)` (or `parseFloat(data)` if it's floating point)?